Cargo.toml

```diff
@@ -10,6 +10,6 @@ categories = ["development-tools"]
 [dependencies]
 rustc-serialize = "0.3"
 log = "0.3"
-rls-data = "= 0.4.1"
+rls-data = { path = "../rls-data", default-features = true, features = ["borrows"] }
```

Comment:
v0.7 is now published, but I think we need to upgrade rustc to use that version before we can make rls-analysis use that version.

Is it necessary to specify `default-features = true`?